cmds/find/find.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Find finds files. It is similar to the Unix command. It uses REs, not globs,
+// for matching.
+//
+// OPTIONS:
+//     -d: enable debugging in the find package
+//     -mode integer-arg: match against mode, e.g. -mode 0755
+//     -type: match against a file type, e.g. -type f will match files
+//     -name: glob to match against file
+//     -l: long listing. It's not very good, yet, but it's useful enough.
+//     -root: where to start the search, default /tmp
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/u-root/u-root/pkg/find"
+)
+
+var (
+	perm      = flag.Int("mode", -1, "Permissions")
+	fileType  = flag.String("type", "", "File type")
+	name      = flag.String("name", "", "glob for name")
+	root      = flag.String("root", "/tmp", "path for root")
```

Comment:
Default root should probably be os.Getwd instead of /tmp.

Reply:
I changed it to be more like the original, it's now a required argument.